geotrek/maintenance/models.py

```diff
@@ -553,7 +553,7 @@ def interventions_csv_display(self):
     @property
     def intervention_contractors(self):
         return self.interventions.aggregate(
-            intervention_contractors=ArrayAgg('contractors__contractor', distinct=True))['intervention_contractors']
+            intervention_contractors=ArrayAgg('contractors__contractor', distinct=True, filter=Q(contractors__contractor__isnull=False)))['intervention_contractors']
```

Comment:
The `Contractor.contractor` fields is not nullable. Why is this filter necessary?

Reply:
Yes it is necessary because contractors Project can be blank and in view detail, if an project have 1 intervention with contractors and an other without contractors, input display "None" like 
![image](https://github.com/GeotrekCE/Geotrek-admin/assets/25923337/b78e7ff8-6308-450f-ad21-b7447ea6f659)